wagtail_wordpress_import/prefilters/transform_styles_defaults.py

```diff
@@ -116,25 +116,21 @@ def conf_styles_mapping():
 
     See the documentation in this repo for further help with creating filters
     """
-    return getattr(
-        settings,
-        "WAGTAIL_WORDPRESS_IMPORT_PREFILTERS",
```

Comment:
It did work until when `WAGTAIL_WORDPRESS_IMPORT_PREFILTERS` was added to settings, when we need to do that for altering the ADDITIONAL_ALLOWED_ATTRIBUTES for bleach clean, it wasn't been included in the in prefilter_content(). In other words we intended that setting to be available for overriding but you actually couldn't. We've not had a use case yet where we needed to do that.

Reply:
Ah, because the original `default_prefilters()` function didn't read settings at all? https://github.com/torchbox/wagtail-wordpress-import/blob/main/wagtail_wordpress_import/importers/wordpress_defaults.py#L4

Reply:
The issue now though is that `conf_styles_mapping()` has a big docstring talking about overriding it, but this too is just returning a non-overrideable constant.

Reply:
If a developer provided

```
WAGTAIL_WORDPRESS_IMPORT_PREFILTERS = [
        ...
        {"FUNCTION": "wagtail_wordpress_import.prefilters.transform_styles_filter",
            "OPTIONS": {
                "TRANSFORM_STYLES_MAPPING": [
                       (regex, method)
                ],
            },
        },
        ...
    ]
```

It would override `conf_styles_mapping()` in `filter_transform_inline_styles()`

We seem to have no tests for that kind of override. Should we do that here or create a ticket for it. It looks like I have a day each fortnight to do work on the package.

Reply:
If it's possible to add the test here, that sounds like a good addition.

Reply:
Hi Nick, adding the test was possible but it has uncovered other issues which I have investigated and can see the fix but's it's a rabbit hole to say the least. Hopefully I can pick this up later after my meeting.

Reply:
Hi Nick, I had to do some refactoring as the new test uncovered some issues.